### PR TITLE
Add parentheses to `EntityManager::contains()`

### PR DESCRIPTION
**Pull request by simPod**
Make order of operations more explicit.

### CODE REVIEW

File: lib/Doctrine/ORM/EntityManager.php [hunk 1]
```diff
@@ -836,8 +836,10 @@ public function getRepository($entityName)
     public function contains($entity)
     {
         return $this->unitOfWork->isScheduledForInsert($entity)
-            || $this->unitOfWork->isInIdentityMap($entity)
-            && ! $this->unitOfWork->isScheduledForDelete($entity);
+            || (
+                $this->unitOfWork->isInIdentityMap($entity)
+                && ! $this->unitOfWork->isScheduledForDelete($entity)
+            );
```
Comment (greg0ire) [inline lib/Doctrine/ORM/EntityManager.php:842]: This is questionable, and should be addressed at the coding standard level, unless for some reason what is not obvious here according to you would be obvious in other places where we combine several boolean operators.

**Comment by derrabus** [comment]
Sorry, but changes like this one are a waste of everybody's time imho.

**Comment by simPod** [comment]
It's possible it's just me, I had to lookup operator precedence to understand the condition. Feel free to close but it caused confusion.

**Comment by greg0ire** [comment]
Sorry but I'm going to close this. Operator precedence between `&&` and `||` is something you should know, really.

**Comment by simPod** [comment]
Keep forgetting what's first 😅

**Pull request closed without merging**